phalcon/http/request.zep

```diff
@@ -127,15 +127,20 @@ class Request implements RequestInterface, InjectionAwareInterface
 	public function getPut(string! name = null, var filters = null, var defaultValue = null, boolean notAllowEmpty = false, boolean noRecursive = false) -> var
 	{
 		var put;
-
-		let put = this->_putCache;
-
-		if typeof put != "array" {
-			let put = [];
-			parse_str(this->getRawBody(), put);
-
-			let this->_putCache = put;
-		}
+        let put = this->_putCache;
+        if typeof put != "array" {
+            if strpos(this->getContentType(), 'json') != false {
```

Comment:
- The `this->getContentType()` may return NULL.
- You have to use "json" instead of 'json'